src/forms/inputs/checkbox-group.js

```diff
@@ -118,22 +150,28 @@ function CheckboxGroup(props) {
     return function (checked) {
       // Add or remove option value from array of values, depending on whether it's checked
       const newValueArray = checked
-        ? addToArray([option.value], value)
+        ? addToArray(value, [option.value])
```

Comment:
Flipped the order so that the new selection shows at the end of the input, rather than at the beginning

Reply:
Good catch! We should add a note about this in the migration guide